com.prateekmedia.appimagepool.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: com.prateekmedia.appimagepool
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
```

Comment:
do you need network?

Reply:
Yes to fetch AppImageHub `feed.json` files and downloading appimages from it.